log_miner.py

```diff
@@ -7,9 +7,10 @@
 ------------------------------------------------------------------------
 """
  
-import cx_Oracle, sys, string, _thread, datetime
+import sys, string, _thread, datetime
```

Comment:
Picky, but this should be separated out into individual ordered import statements, helps with git diffing - now I know why people do that!

Reply:
P.S. This appears in a few of the "tests" files too

Reply:
Agreed, I didn't touch the original code to much - but you are right - it certainly wouldn't pass a Linter (there is an opportunity to make it better)

Reply:
Done run the import statements through isort.